.travis.yml

```diff
@@ -16,3 +16,6 @@ deploy:
     master: orionchess
   on:
     repo: FirehoseProjectTeamOrion/OrionChess
+addons:
+  code_climate:
+    repo_token: #Token Goes Here
```

Comment:
Could you add the token. I think you should have access to travis & be able to add our repo to codeclimate (auth is with github). Let me know if you need help :)